app/Rest/Resources/PostResource.php

```
@@ -46,6 +46,7 @@ public function relations(RestRequest $request): array
         return [
             BelongsTo::make('user', UserResource::class),
             BelongsTo::make('category', CategoryResource::class),
+            BelongsToMany::make('user_exploited', UserResource::class),
```

PHPMD (2.15.0)

Avoid using static access to class '\Lomkit\Rest\Relations\BelongsToMany' in method 'relations'. (Clean Code Rules)

(StaticAccess)